core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -1150,7 +1151,11 @@ public static String dnsCacheSpec() {
      * to override the default value.
      *
      * @see ExceptionVerbosity
+     *
+     * @deprecated Use {@link LoggingService} or log exceptions using
+     *             {@link ServerBuilder#exceptionHandler(com.linecorp.armeria.server.ExceptionHandler)}.
      */
+    @Deprecated
```

Comment:
We don't have to deprecate this anymore because it's actually used?

Reply:
According to this discussion(https://github.com/line/armeria/pull/3674#discussion_r676595119), 
I thought we decided not to recommend using `ExceptionVerbosity`. I wonder `annotatedServiceExceptionVerbosity` is useful in the real world.
@KarboniteKream @ks-yim Any thought on this?

Reply:
> According to this discussion(#3674 (comment)),

Isn't it about the `ExceptionVerbosity` that we have tried to newly introduce in this PR?
Users can achieve the same behavior using `exceptionHandler` so we revert it.
However, Isn't `annotatedServiceExceptionVerbosity` different? At least, it's used some places so I wonder if we can remove it someday. 🤔

Reply:
Perhaps, we can remove it Armeria 2.0.
I thought it is awkward to support exception verbosity only for annotated service. That was the reason I added exception verbosity to a `SeverBuilder`

Reply:
Had a chat with @ikhoon and we decide to remove this from Armeria 2.0
because users can check the exception using `LoggingSerivce` by this change: https://github.com/line/armeria/pull/3674/commits/39f25996291727edb181e3e3750f2e1a2f739288